### PR TITLE
chore: update charts to v0.2.18

### PR DESCRIPTION
**Pull request by skyerus**
## This PR

Update charts to v0.2.18.
Result of running `make helm-package`.

### CODE REVIEW

File: chart/templates/rendered.yaml [hunk 1]
```diff
@@ -544,7 +813,7 @@ spec:
         env:
         - name: FLAGD_VERSION
           value: v0.2.5
-        image: ghcr.io/open-feature/open-feature-operator:main
+        image: controller:latest
```
Comment (beeme1mr) [inline chart/templates/rendered.yaml:816]: Can you explain why this change is being made?

Comment (skyerus) [reply chart/templates/rendered.yaml:816]: This was unintentional, reverting

**Review by beeme1mr — APPROVED** [review]
